### ISSUE TITLE
Outputs (inside the "resources" section) are not merged together

### ISSUE THREAD

**Issue by LoZeno** [icarus-sullivan/serverless-plugin-modularize#3]
Apologies for the 1-2 of issues raised - I normally don't use Outputs but there's always the one rogue project that does things differently from the rest of the stack

This is definitely related to issue #2 : using version 1.0.6 (thanks for the fix BTW, really appreciated) now the `resources` section of the resulting merged yaml contains the correct `Resources`, but no `Outputs`
See the bottom of this reference for an example of `Outputs`:
https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/

I think the fix would be the same thing done for `Resources` in [merge.js](https://github.com/icarus-sullivan/serverless-plugin-modularize/blob/f41714b12eb17fb7bbca444ac0bfd4ff564402b3/src/utils/merge.js#L17)
```javascript
  resources: {
    Resources: {
      ...get(a, 'resources.Resources', {}),
      ...get(b, 'resources.Resources', {}),
    },
    Outputs: {
      ...get(a, 'resources.Outputs', {}),
      ...get(b, 'resources.Outputs', {}),
    },
```

**Comment by icarus-sullivan** [comment]
You're right, I shouldn't have been so quick to close the previous issue. I'll get a fix out soon enough and update you.

**Comment by icarus-sullivan** [comment]
Once again, I appreciate your feedback and help. 

I published 1.0.7 that includes the Outputs. I'm going to open a different Issue regarding some additional missing properties. 

- package
- layers 
- ... maybe more

**Comment by icarus-sullivan** [comment]
https://github.com/icarus-sullivan/serverless-plugin-modularize/issues/4
